testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveStorageFormats.java

```diff
@@ -391,7 +392,7 @@ public void testInsertAndSelectWithNullFormat(StorageFormat storageFormat)
 
         assertThat(query(format("SELECT * FROM %s", tableName))).containsOnly(storedValues);
 
-        onHive().executeQuery(format("DROP TABLE %s", tableName));
+        query(format("DROP TABLE %s", tableName));
```

Comment:
The change here is correct, but should be separate commit, titled something like

```
Drop table using same executor it was created with
```

Reply:
without this change this particular test cannot be run in HMS_ONLY suite, this is why I think it suits in this commit